src/components/popovers/Menu.jsx

```diff
@@ -1,19 +1,25 @@
 import React from 'react';
+import cx from 'classnames';
 import PropTypes from 'prop-types';
 import {
   Menu as ReachMenu,
-  MenuList as ReachMenuList,
+  MenuItems as ReachMenuItems,
   MenuButton as ReachMenuButton,
+  MenuPopover as ReachMenuPopover,
 } from '@reach/menu-button';
 
 const Menu = ({ isInModal = false, children, trigger }) => (
-  <ReachMenu className={isInModal && 'fdsMenu--inModal'}>
+  <ReachMenu>
     <ReachMenuButton className="resetButton border--focus rounded--all">
       {trigger}
     </ReachMenuButton>
-    <ReachMenuList className="elevation--2 rounded--all bgColor--white">
-      {children}
-    </ReachMenuList>
+    <ReachMenuPopover
```

Comment:
What's the reason for opting for `MenuPopover` instead of `MenuList`? I see in the docs that `MenuPopover` allows more control.

Reply:
`MenuPopover` gives us direct access to the `className` property of the containing popover